src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -20,5 +20,10 @@
             <argument type="service" id="fos_oauth_server.entity_manager" />
             <argument>%fos_oauth_server.model.client.class%</argument>
         </service>
+
+        <service id="sylius.listener.api.add_to_cart" class="Sylius\Bundle\ApiBundle\EventListener\AddToCartListener">
+            <argument type="service" id="sylius.order_processing.order_processor.composite" />
+            <tag name="kernel.event_listener" event="sylius.order_item.post_create" method="cartItemResolver" />
```

Comment:
event `pre_create`
method `recalculateOrder`?

Shouldn't we recalculate order on update also?